app/src/main/res/layout/return_to_lesson_button_item.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="buttonViewModel"
+      type="org.oppia.android.app.hintsandsolution.ReturnToLessonViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Probably don't need this since there's just one element. Also, we aren't setting a width/height on the button.

Reply:
Updated and removed `ConstraintLayout`